jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -745,7 +724,6 @@ public OrderedJsonObject getDebugState(String conferenceId, String endpointId, b
         debugState.put("time", System.currentTimeMillis());
 
         debugState.put("health", getHealthStatus());
-        debugState.put("e2e_packet_delay", Endpoint.getPacketDelayStats());
```

Comment:
Did you mean to remove this?

Reply:
Yes. It is still available under `/debug/stats/transit-stats`. I don't think this is a good place for it, because it makes the output of `/debug` quite long (and I use `/debug` all the time to get the conference IDs).